code/game/gamemodes/miniantags/demons/shadow_demon/shadow_demon.dm

```diff
@@ -6,6 +6,7 @@
 	icon_living = "shadow_demon"
 	move_resist = MOVE_FORCE_STRONG
 	lighting_alpha = LIGHTING_PLANE_ALPHA_MOSTLY_VISIBLE // so they can tell where the darkness is
+	see_in_dark = 10
```

Comment:
I would leave a comment explaining why this needs to be ten

Reply:
black corners i mentioned earlier in the dark. But yes, this is worth explaining in code